### PR TITLE
Fix the onnx symbolic for elu and maxpool3d

### PR DESCRIPTION
**Pull request by houseroad**
They were broken... just fixed.

### CODE REVIEW

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -395,7 +395,7 @@ def max_pool1d(g, input, kernel_size, stride, padding, dilation, ceil_mode):
 def max_pool2d(g, input, kernel_size, stride, padding, dilation, ceil_mode):
     if ceil_mode:
         return _unimplemented("max_pool2d", "ceil_mode")
-    if set(_pair(dilation)) != {1}:
+    if set(_pair(dilation)) != {1, 1}:
```
Comment (bddppq) [inline torch/onnx/symbolic.py]: {1, 1} is equivalent to {1}

File: torch/onnx/symbolic.py [hunk 2]
```diff
@@ -406,6 +406,20 @@ def max_pool2d(g, input, kernel_size, stride, padding, dilation, ceil_mode):
     return r, None
 
 
+def max_pool3d(g, input, kernel_size, stride, padding, dilation, ceil_mode):
+    if ceil_mode:
+        return _unimplemented("max_pool3d", "ceil_mode")
+    if set(_triple(dilation)) != {1, 1, 1}:
```
Comment (ezyang) [inline torch/onnx/symbolic.py]: Ditto here... I think you didn't want `set()` here?

**Review by ezyang — DISMISSED** [review]
see comment

**Comment by houseroad** [comment]
ping @ezyang for review :-)

**Review by ezyang — APPROVED** [review]

**Pull request merged**